server/views/pages/allocate-to-practitioner.njk

```diff
@@ -34,6 +34,9 @@
         <div class="govuk-grid-column-two-thirds">
             <h1 class="govuk-heading-xl govuk-!-margin-bottom-10">
         You're allocating {{name}} ({{tier}}) to {{ offenderManagerDetails }}</h1>
+            {% if 'true' == laoCase %}
```

Comment:
Are if 'true'  and if true without the apostophe truly interchangeable both have used? I had problems on some of my tests when I made the mockApi return 'false' instead of false. @iainjchambers-moj